dns-metrics.schema.json

```diff
@@ -171,9 +171,160 @@
                             }
                         }
                     }
+                },
+                "conn_info": {
+                    "$ref": "#/$defs/conn_info"
                 }
             },
             "required": [ "since", "until" ]
+        },
+        "udp_conn": {
+            "title": "DNS Metric UDP Connection Information",
+            "description": "An object with the metrics around the UDP connections used to sent and received for the given stats object",
+            "type": "object",
+            "properties": {
+                "packets_sent": {
+                    "description": "The number of packets sent",
+                    "type": "integer"
+                },
+                "packets_received": {
+                    "description": "The number of packets received",
+                    "type": "integer"
+                },
+                "socket_errors": {
+                    "description": "The number of various socket errors that occurred",
+                    "type": "integer"
+                }
+            }
+        },
+        "tcp_conn": {
+            "$ref": "#/$defs/udp_conn",
+            "title": "DNS Metric TCP Connection Information",
+            "description": "An object with the metrics around the TCP connections used to sent and received for the given stats object",
+            "properties": {
+                "handshakes": {
+                    "description": "The number of handshakes performed",
```

Comment:
performed = successful? or attempted?

Reply:
:shrug: this is cut&paste from @nicki-krizek 's comment

Reply:
Thread migrated to: https://github.com/DNS-OARC/dns-metrics/issues/11